doc/policy/packages.md

```diff
@@ -18,7 +18,7 @@ tip or some preceding transaction in the package.
 
 The following rules are enforced for all packages:
 
-* Packages cannot exceed `MAX_PACKAGE_COUNT=25` count and `MAX_PACKAGE_SIZE=101KvB` total size
+* Packages cannot exceed `MAX_PACKAGE_COUNT=25` count and `MAX_PACKAGE_WEIGHT_KWU=404` total weight
```

Comment:
66f6868c4331d3c289221441130e6c1fb61bc60c

this says `MAX_PACKAGE_WEIGHT_KWU` but the variable is named `MAX_PACKAGE_KWU`

Reply:
fixed